public/css/index.css

```diff
@@ -85,3 +85,82 @@
     justify-content: center;
     overflow-x: hidden;
 }
+
+/*........... BLOG SECTION.......... */
+#post-container {
+  padding: 0 25px;
+  max-width: 1400px;
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-rows: 1fr 1fr;
```

Comment:
This enforces a fixed size of two columns. This becomes problematic when we don't have enough blog posts to fill both columns: it forces the grid to take up the remaining space even though there are not blog posts to show. 
I think this should work:
```css
  grid-template-rows: repeat(auto-fill, 1fr);
```